hardhat/hardhat.config.ts

```diff
@@ -7,10 +7,26 @@ import 'hardhat-deploy'
 import * as dotenv from 'dotenv'
 
 import {
-  ACCOUNT_ADDRESSES,
-  getAccount,
+  loadAddress,
+  loadPrivateKey,
+  getAllKeys,
+  setAccount,
+  getAllAddresses,
+  ACCOUNTS
 } from './utils/accounts'
 
+//override default publically known keys by doppler config
+for (const acc of ACCOUNTS) {
+  if (process.env[acc.name.toUpperCase()+'_PRIVATE_KEY'] && process.env[acc.name.toUpperCase()+'_ADDRESS']) {
+    acc.address = loadAddress(acc.name, process.env[acc.name+'_ADDRESS']?.toString() ?? '')
+    acc.privateKey = loadPrivateKey(acc.name, process.env[acc.name+'_PRIVATE_KEY']?.toString() ?? '')
+    setAccount(acc)
+  }
+}
```

Comment:
How does this improve the existing mechanism for loading from the environment? The `ACCOUNTS` constant calls `loadAddress` and `loadPrivateKey` when it is created, for example:

https://github.com/Lilypad-Tech/lilypad/blob/14c3dc9432dbbeced4ce93aebd7ba1d244d4726c/hardhat/utils/accounts.ts#L23-L28